linux_os/guide/system/accounts/accounts-physical/require_singleuser_auth/ansible/shared.yml

```diff
@@ -0,0 +1,25 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{% if init_system == "systemd" -%}}
+- name: require single user mode password
+  lineinfile:
+      create: yes
+      dest: /usr/lib/systemd/system/rescue.service
+      regexp: "^#?ExecStart="
+      {{% if product == "fedora" -%}}
+      line: "ExecStart=-/usr/lib/systemd/systemd-sulogin-shell"
```

Comment:
The corresponding Bash remediation (`in linux_os/guide/system/accounts/accounts-physical/require_singleuser_auth/bash/shared.sh`) adds also `rescue` on this line.